capsules/core/src/virtualizers/virtual_i2c.rs

```diff
@@ -307,9 +398,17 @@ impl<'a, I: i2c::I2CMaster<'a>> i2c::I2CDevice for I2CDevice<'a, I> {
     }
 }
 
+impl<'a, I: i2c::I2CMaster<'a>> i2c::I2CMultiDevice for I2CDevice<'a, I, true, NoSMBus> {
```

Comment:
Why not just use `I2CMaster`?

Reply:
I feel the layering is a little weird using `I2CMaster`. As the `I2CDevice` layers on top of `I2CMaster` it feels a little cleaner to then also layer the "rest of the bus" on `I2CMaster` as well.

To me it also makes it clearer that we only have access to some of the bus, instead of full access that `I2CMaster` has.

By using `I2CMultiDevice` we can piggy back on the existing `I2CDevice` support. Plus then we need to check the address less often, only when setting it instead of on every operation